[2022-12-16 11:04:06,246] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-12-16T02:07:00+00:00 [queued]>
[2022-12-16 11:04:06,365] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-12-16T02:07:00+00:00 [queued]>
[2022-12-16 11:04:06,382] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-16 11:04:06,382] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-16 11:04:06,382] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-16 11:04:06,514] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_last_id> on 2022-12-16 02:07:00+00:00
[2022-12-16 11:04:06,526] {standard_task_runner.py:52} INFO - Started process 3475 to run task
[2022-12-16 11:04:06,561] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'prepare_data', 'get_last_id', 'scheduled__2022-12-16T02:07:00+00:00', '--job-id', '1736', '--raw', '--subdir', 'DAGS_FOLDER/prepare_data.py', '--cfg-path', '/tmp/tmp9ovbm9dg', '--error-file', '/tmp/tmpd9vu659k']
[2022-12-16 11:04:06,573] {standard_task_runner.py:80} INFO - Job 1736: Subtask get_last_id
[2022-12-16 11:04:06,921] {task_command.py:369} INFO - Running <TaskInstance: prepare_data.get_last_id scheduled__2022-12-16T02:07:00+00:00 [running]> on host 8e2f65ec4c74
[2022-12-16 11:04:07,558] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=prepare_data
AIRFLOW_CTX_TASK_ID=get_last_id
AIRFLOW_CTX_EXECUTION_DATE=2022-12-16T02:07:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-16T02:07:00+00:00
[2022-12-16 11:04:07,634] {base.py:68} INFO - Using connection ID 'gp_conn' for task execution.
[2022-12-16 11:04:07,817] {python.py:173} INFO - Done. Returned value was: None
[2022-12-16 11:04:08,061] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=prepare_data, task_id=get_last_id, execution_date=20221216T020700, start_date=20221216T110406, end_date=20221216T110408
[2022-12-16 11:04:08,185] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-16 11:04:08,435] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
